ansible/module_utils/port_utils.py

```diff
@@ -196,7 +196,7 @@ def get_port_alias_to_name_map(hwsku, asic_name=None):
         elif hwsku == "64x100Gb":
             for i in range(0, 64):
                 port_alias_to_name_map["Ethernet%d" % i] = "Ethernet%d" % i
-        elif hwsku == "8800-LC-48H-O":
+        elif hwsku in ["8800-LC-48H-O", "88-LC0-36FH-MO"]:
```

Comment:
could use HWSKU_WITH_PORT_INDEX_FROM_PORT_CONFIG ?

Reply:
@SuvarnaMeenakshi `elif` has `dict` for alias to name mapping which is speciifc to this SKKU's. Going forward HWSKU_WITH_PORT_INDEX_FROM_PORT_CONFIG can expand to other sku's where this `elif` condition might not be applicable